core/src/main/java/org/apache/calcite/adapter/enumerable/EnumerableLimitRule.java

```diff
@@ -30,7 +31,7 @@
 class EnumerableLimitRule extends RelOptRule {
   EnumerableLimitRule() {
     super(
-        operand(Sort.class, any()),
+        operand(LogicalSort.class, any()),
```

Comment:
Please update the javadoc accordingly (`org.apache.calcite.rel.logical.LogicalSort` instead of `org.apache.calcite.rel.core.Sort`). The same for the rest of the modified rules.

Reply:
Good catch. Fixed. Thx.